ftwhttp/request.go

```diff
@@ -140,28 +147,21 @@ func BuildRequest(r *Request) ([]byte, error) {
 		r.AddStandardHeaders()
 	}
 
-	_, err = r.Headers().WriteBytes(&b)
+	err = r.Headers().Write(&b)
 	if err != nil {
 		log.Debug().Msgf("ftw/http: error writing to buffer: %s", err.Error())
 		return nil, err
 	}
 
-	// TODO: handle cookies
-	// if client.Jar != nil {
-	// 	for _, cookie := range client.Jar.Cookies(req.URL) {
-	// 		req.AddCookie(cookie)
-	// 	}
-	// }
-
 	// After headers, we need one blank line
-	_, err = fmt.Fprintf(&b, "\r\n")
+	_, err = b.WriteString("\r\n")
```

Comment:
Same here? We have a constant for headerSeparator, maybe we want to use it here? https://github.com/coreruleset/go-ftw/pull/443/files#diff-fad61aedc5779df83ed3dfea5f520c19e259eafc9e86bdf09447d7dab9706e65R18